tests/Tests/500__jupyterhub/minimal-cuda-test.robot

```diff
@@ -45,6 +45,14 @@ Verify Tensorflow Library Can See GPUs In Minimal CUDA
     ...     ODS-1143
     Verify Tensorflow Can See GPU    install=True
 
+Verify Cuda Image Have NVCC Installed
+    [Documentation]     Verifies NVCC Version in Minimal CUDA Image
+    [Tags]  Sanity
+    ...     ODS-483
+    ${nvcc_version} =  Run Cell And Get Output    input=!nvcc --version| grep "release"| awk '{print $5}'
+    ${nvcc_version} =  Fetch From Left    ${nvcc_version}  ,
+    Should Be Equal As Strings    ${nvcc_version}  ${EXPECTED_CUDA_VERSION}
```

Comment:
Is this actually checking the nvcc version or just the CUDA version?
If it's checking the CUDA version (given the comparison is done against `${EXPECTED_CUDA_VERSION}`), this is already done in `Verify Installed CUDA Version` in the `Verify CUDA Image Includes Expected CUDA Version` TC

Reply:
It must check if nvcc is installed or not.